backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -335,10 +316,35 @@ public RepositoryEntity getRepositoryById(Session session, RepositoryIdentificat
             session, commitEntity.getCommit_hash(), ModelDBConstants.MASTER_BRANCH, repository);
       }
       session.getTransaction().commit();
+      if (create) {
```

Comment:
1. once https://github.com/VertaAI/modeldb/pull/649 is in we should create the repo with delete flag true.
1. create the role binding
    1. if the role binding creation succeeds, update the delete flag
    1.if the role binding creation fails, return error.

Reply:
I will update this logic after merge #649 .

Reply:
Please file a ticket so that we can track it, refer this comment so we have context

Reply:
https://vertaai.atlassian.net/browse/VR-4406